app/src/scenes/inscription/Home/index.js

```diff
@@ -11,14 +11,23 @@ import DesktopView from "./DesktopView";
 export default function Home({ location }) {
   const history = useHistory();
 
+  const registerEventPlausibleMobile = (e, props) => {
+    // eslint-disable-next-line no-undef
+    plausible(e, { ...props, device: "mobile" });
```

Comment:
```suggestion
    window.plausible?.(e, { ...props, device: "mobile" });
```

Pour éviter le "no undef" et pour ne pas planter dans le cas où il y a un bloqueur de traceur.